### DISCUSSION TITLE
[vscode|powershell] Getting recommendation in a completly wrong language

### DISCUSSION THREAD

**Discussion by COLAMAroro** [community/community#8170] [Copilot Conversations]
Hello !

I've wanted to throw a quick PowerShell script to get some infos about my music library, including the total duration

Unless PowerShell's syntax changed drastically overnight to look like ~~Python~~ Ruby, GitHub Copilot seems to get the syntax from a completely different language pool

![Screenshot of the issue](https://i.imgur.com/YEQqfbc.png)

**Comment by jdhitsolutions** [comment]
I am having the same experience. Just installed the Copilot extension in the current release of VSCode. No matter what I try, I keep getting what looks like bash or linux suggestions.

**Comment by VOVELEE** [comment]
I have the same experience. Looks PowerShell is not supported by GitHub Copilot
